yugabyte/src/yugabyte/multi_key_acid.clj

```diff
@@ -56,15 +56,16 @@
 
 (defn workload
   [opts]
-  (let [n (count (:nodes opts))]
+  (let [n (count (:nodes opts))
+        threads (:concurrency opts)]
     {:generator (ygen/with-op-index
                   (independent/concurrent-generator
-                    (* 2 n)
+                    (/ threads 2)
                     (range)
```

Comment:
Shouldn't this be reworked just like SKA was?

Reply:
Nope, multi-key-acid has other logic - we are able to insert values, if it's absent.